integration/sawtooth_integration/docker/two-families.yaml

```diff
@@ -23,8 +23,6 @@ services:
       - $SAWTOOTH_CORE:/project/sawtooth-core
     expose:
       - 40000
-    depends_on:
```

Comment:
These 'depends_on' deletions shouldn't be part of this PR. I mentioned in RocketChat that I had spread some bad information about needing these statements in the docker compose yaml files: https://chat.hyperledger.org/channel/sawtooth?msg=cM4rrDvj95WfaPrrD. We should verify that all of our integration tests can pass reliably without depends_on, and then make a PR that removes all of them.